package.json

```diff
@@ -20,6 +20,7 @@
     "build:main": "cross-env NODE_ENV=production webpack --bail --config webpack.config.main.js",
     "build:renderer": "cross-env NODE_ENV=production webpack --bail --config webpack.config.renderer.js",
     "start": "electron src --disable-dev-mode",
+    "clean": "rm -rf dist/ release/ node_modules/ src/node_modules/ build/ src/*_bundle.js src/**/*_bundle.js",
```

Comment:
Is `rm -rf dist/ release/ node_modules/ src/node_modules/ build/ && find src -name '*_bundle.js' | xargs rm` appropriate for you? This can remove `src/browser/webview/mattermost_bundle.js`

Reply:
@dmeza What about this?

Reply:
@yuya-oc changed and tested.